src/EventListener/TagSubscriber.php

```diff
@@ -73,4 +74,15 @@ protected function isShopContext(Request $request = null): bool
 
         return $firewallConfig->getName() === 'shop';
     }
+
+    protected function isMainRequest(RequestEvent $event): bool
+    {
+        if (method_exists($event, 'isMainRequest')) {
+            return $event->isMainRequest();
+        }
+
+        // BC Layer for Symfony < 5.3
+        /** @psalm-suppress DeprecatedMethod */
```

Comment:
Add it to Psalm config instead

Reply:
How? (+ I actually think it's nice to have this comment to make it easy to drop the bc layer later)

Reply:
Yeah, I think you will gain the same benefits of just removing it from Psalm config, but let's just keep it :)